source/linux/Foundational_Components_Multimedia_wave5.rst

```diff
@@ -630,27 +975,43 @@ Buffer import on decoder can be tested by setting the ``capture-io-mode`` to ``5
 Configuration of CMA Size
```

Comment:
This feels like a generic how-to guide. It's not really multimedia specific. May be worth separating at some point.

Reply:
Valid point, I can do this after current release is finished